WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -54,6 +54,15 @@ local controlPointFunctions = {
   ["ClearAnchorPoint"] = function(self)
     self.point, self.relativeFrame, self.relativePoint, self.offsetX, self.offsetY = nil, nil, nil, nil, nil
   end,
+  ["ReAnchor"] = function(self, point)
```

Comment:
The call of Reanchor doesn't include a point.

Reply:
It does in WeakAuras.lua, the ReAnchor() you find in DynamicGroup.lua is a method of moversizer's frame